packages/normalizr/src/entities/Entity.ts

```diff
@@ -63,6 +66,39 @@ export default abstract class Entity extends SimpleRecord {
     return this.prototype.pk.call(value, parent, key) || key;
   }
 
+  /** Creates new instance copying over defined values of arguments */
+  static merge<T extends typeof Entity>(
+    this: T,
+    existing: Partial<AbstractInstanceType<T>>,
+    incoming: Partial<AbstractInstanceType<T>>,
+  ) {
+    return { ...existing, ...incoming };
+  }
```

Comment:
this seems like an odd thing to add given that it's so simple and idiomatic to do it inline

Reply:
this allows user customization. https://resthooks.io/docs/api/Entity#static-mergeexisting-incoming-mergedvalue

For instance, coinbase's prices were all wrong until we used timestamps to resolve which

Reply:
ah, meaning that the base functionality is just an object spread, but user entities might want to override it. fair enough

Reply:
If this was merely a helper to simplify code it would be private